connector/saml/saml.go

```diff
@@ -252,6 +281,11 @@ func (p *provider) POSTData(s connector.Scopes, id string) (action, value string
 		},
 		AssertionConsumerServiceURL: p.redirectURI,
 	}
+
+	if p.responseBinding == connector.SAMLBindingRedirect {
+		r.ProtocolBinding = bindingRedirect
```

Comment:
Do we not set this if it's a POST?

Reply:
It's already set to POST in a few lines above when constructing the authnRequest.